components/AddMemory.js

```diff
@@ -12,20 +12,23 @@ const AddMemory = ({ handleSubmit, text, setText, song, setSong }) => {
                     <Row className="justify-content-center">
                         <Form.Group controlId="formSong">
                             <Form.Label>Write a song name</Form.Label>
-                            <Form.Control 
-                                as="textarea" 
-                                onChange={({target}) => setSong(target.value)} 
-                                rows="1" 
+                            <Form.Control
+                                as="textarea"
+                                onChange={({ target }) => {
+                                    setSong(target.value)
+                                    handleAutocomplete()
```

Comment:
Don't think we need this any more if you're using the useeffect thing. This might be causing the error